// Copyright 2020 The Chromium Authors
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.

syntax = "proto2";

package feedwire;

option optimize_for = LITE_RUNTIME;

message ContentId {
  enum Type {
    TYPE_UNDEFINED = 0;
    CLUSTER = 3;
    FEATURE = 4;
  }
  optional string content_domain = 1;
  optional int32 type = 2;
  optional fixed64 id = 3;
}
